yk_utils/files/configuration_files.py

```diff
@@ -0,0 +1,9 @@
+"""Handling of configuration files"""
+import json
+
+
+def load_json_config(filename='./client_secrets.json'):
```

Comment:
Can we remove this default and throw ValueError is filename is empty?

Reply:
typing missing in this method.

Reply:
Sure, will fix this :)